vpr/src/timing/PreClusterDelayCalculator.h

```diff
@@ -165,7 +163,7 @@ class PreClusterDelayCalculator : public tatum::DelayCalculator {
     const AtomNetlist& netlist_;
     const AtomLookup& netlist_lookup_;
     const float inter_cluster_net_delay_;
-    const std::unordered_map<AtomBlockId, t_pb_graph_node*> block_to_pb_gnode_;
+    const Prepacker& prepacker_;
```

Comment:
Tatum is a git submodule so this should be pushed back to its repo too when you're done.

Reply:
You are correct, I thought the same exact thing; however, this is actually still part of VTR. Its in src/timing. This code actually inherits from the Tatum DelayCalculator to prevent changing upstream Tatum. This was actually very well done, whoever made this! So I do not think Tatum needs to be updated in this case.

Reply:
Ah, got it!  Kevin Murray created Tatum, and you're right the delay calculators were intentionally left as callbacks into client code.